### ISSUE TITLE
Can i create namespaces (like a socket) for different rooms for users?

### ISSUE THREAD

**Issue by mencerz** [lance-gg/lance#89]

**Comment by namel** [comment]
There is no such feature at the moment.  Just to make sure I understand the request:  you want to create a namespace per room, so that the sync broadcasts for a specific room will only go to players who are in that room?

**Comment by mencerz** [comment]
Exactly! Thanks for the quick response.

**Comment by mencerz** [comment]
So, what i can do?

**Comment by elr0berto** [comment]
Sorry if this question is inappropriate. But I've been considering games with massive amounts (thousands) of players.
The way I imagine it works is to separate the game world into "areas" or "rooms" or "namespaces" .. is this what you are talking about?

Or
Are you talking about multiple instances of the same game?

**Comment by namel** [comment]
This particular github issue is about "rooms", where different players connect to a single server instance, and will only see a subset of what that single server instance is controlling (one room).  It's not about clustering the game to multiple server instances.

**Comment by namel** [comment]
This is implemented in [lance-gg@3.1.0](https://www.npmjs.com/package/lance-gg).  New API is:
```javascript
ServerEngine.createRoom(roomName)
ServerEngine.assignObjectToRoom(obj, roomName)
ServerEngine.assignPlayerToRoom(playerId, roomName)

```
New event:
```
client___roomUpdate
```

see [docs](http://docs.lance.gg/ServerEngine.html#createRoom)
